libs/packages/components/src/lib/tab-outside/taboutside.directive.ts

```diff
@@ -14,8 +14,20 @@ export class SDSTabOutsideDirective {
   @HostListener('document:keyup', ['$event.target'])
   public hasFocusChanged(target) {
     const isInsideHost = this._elementRef.nativeElement.contains(target);
-    if (!isInsideHost) {
+    const isAutocomplete = this.hasClass(target, 'sds-autocomplete');
```

Comment:
What are the changes in this file supposed to do? I'm trying to see a difference when testing against an existing autocomplete demo and not seeing any difference in behavior when tabbing around.

Reply:
reverted back to original, it is the same functionality